### DISCUSSION TITLE
is there any code or way to find gaze detection (seeing left, right, straight) by using pupil of eyes, using dlib c++ ,any references

### DISCUSSION THREAD

**Discussion by ranjith502** [davisking/dlib#2792] [Q&A]
any help

**Comment by davisking** [comment]
Not in dlib.  You would need to build that using the available lower level tools.

**Reply by ranjith502** [reply]
@davisking ,what is mean by available lower level tools.

**Comment by davisking** [comment]
I mean you have to code it yourself :)

**Comment by ranjith502** [comment]
thanks bro ,i will look into it
